src/app/login/login.js

```diff
@@ -1,19 +1,19 @@
-var login = `
-<div class="login-container">
-  <div class="login-text-box">
-    <h3>Login</h3>
-    <p style="font-size: 80%;">Get access to your Orders, Wishlist and Recommendations</p>
-  </div>
-  <form class="login-form">
-    <div class="wrap-input100">
-      <input class="input100" type="text" name="email">
-      <span class="focus-input100" data-placeholder="Email"></span>
-    </div>
-    <div class="wrap-input100">
-      <input class="input100" type="password" name="pass">
-      <span class="focus-input100" data-placeholder="Password"></span>
+export const login = `
+  <div class="login-container">
+    <div class="login-text-box">
+      <h3>Login</h3>
+      <p style="font-size: 80%;">Get access to your Orders, Wishlist and Recommendations</p>
     </div>
-    <a class="btn-login" href="#">Login</a>
-  </form>
-</div>
+    <form class="login-form">
```

Comment:
Why does your form doesn'r have a post="method" attribute??? Explain me why its important and should always be used in sensitive forms.